src/shared/components/network.stories.tsx

```diff
@@ -0,0 +1,162 @@
+import type { Meta, StoryObj } from "@storybook/react-vite";
+import { Network } from "./network";
+import { publicProvider, StarknetConfig } from "@starknet-react/core";
+import { sepolia, mainnet, Chain } from "@starknet-react/chains";
+import { constants, num, shortString } from "starknet";
+import { STRK_CONTRACT_ADDRESS } from "@cartridge/utils";
+
+const meta = {
+  tags: ["autodocs"],
+  title: "Network Button",
+  component: Network,
+} satisfies Meta<typeof Network>;
+
+// minimal setup for starknet provider to simulate different networks
+const StarknetProvider = ({
+  children,
+  chainID,
+}: {
+  children: React.ReactNode;
+  chainID: bigint;
+}) => {
+  const slotChain: Chain = {
+    id: num.toBigInt(shortString.encodeShortString("WP_SLOT")),
```

Comment:
```suggestion
    id: num.toBigInt(shortString.encodeShortString("WP_JOKERSOFNEONALPHA")),
```

let's include both short and long Slot ids

Reply:
need to demonstrate the truncated version